env_vars.sh

```diff
@@ -3,12 +3,16 @@
 #export GIGASPEECH_RELEASE_URL=oss://speechcolab/GigaSpeech/release/GigaSpeech
 
 # Distribution Channel 2: Tsinghua Host
-export GIGASPEECH_RELEASE_URL=http://www.tsinghua-ieit.com/dataset/GigaSpeech
+#export GIGASPEECH_RELEASE_URL=http://www.tsinghua-ieit.com/dataset/GigaSpeech
 
 # Distribution Channel 3: Haitian Host
+#export GIGASPEECH_RELEASE_URL=
 
 # Distribution Channel 4: MagicData Host
 
 # Distribution Channel 5: From IPFS
+declare -A GIGASPEECH_RELEASE_URL
+GIGASPEECH_RELEASE_URL=([tsinghua]='http://www.tsinghua-ieit.com/dataset/GigaSpeech' [speechocean]='124.207.81.184')
```

Comment:
Change it to something like:
```bash
GIGASPEECH_RELEASE_URL=(
  [tsinghua]='http://www.tsinghua-ieit.com/dataset/GigaSpeech'
  [speechocean]='124.207.81.184')
```
So it will be easier to add new lines?